src/components/activity-edit-form/ActivityEditForm.tsx

```diff
@@ -211,7 +184,9 @@ const ActivityEditForm = ({ onClose, activity }: ActivityEditFormProps) => {
                             max={50}
                             placeholder="What's the name of your activity?"
                             value={title}
-                            onChange={handleTitleChange}
+                            onChange={(event: ChangeEvent<TextFieldElement>) =>
+                                setTitle(event.target.value)
+                            }
```

Comment:
I actually meant that you'd leave the ones that take event as their own functions